repo/packages/S/sysdig-cloud/1/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "assets": {
+    "container": {
+      "docker": {
+        "agent": "ceizner/agent:latest"
```

Comment:
please use a tagged version of the image and not latest.

Reply:
> We tag all our agent images as latest. Otherwise, use would need to get new
Mesos package every time we release an agent
We ask that the image be tagged is so that we know what version of base tech works with what version of DC/OS. Using latest can cause broken packages.